src/components/Counter/index.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react'
+import './CounterStyles.css'
+import Button from '../Button'
+
+class Counter extends Component {
+
+  state = {
+    count: 0
+  }
+
+  render() {
+    const increment = () => {
+      this.setState({ count: this.state.count + 1 })
+    }
+    const decrement = () => {
+      this.setState({ count: this.state.count - 1 })
+    }
+
+    return (
+        <div className="counterContainer">
+            <Button click={increment} className='standardButton' iconClassName='icon-angle-up' />
```

Comment:
just to be consistent - let's use double quotes for all attributes
```suggestion
        <div className="counterContainer">
            <Button click={increment} className="standardButton" iconClassName="icon-angle-up" />
```

Reply:
comment Solved